scripts/genDTS.pl

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
```

Comment:
I think its accidentally changed, please fix it, and also please check other scripts that pushed in this PR

Reply:
Done